src/main/scala/cognite/spark/v1/PushdownUtilities.scala

```diff
@@ -24,6 +31,28 @@ object PushdownUtilities {
   def pushdownToUri(parameters: Seq[Map[String, String]], uri: Uri): Seq[Uri] =
     parameters.map(params => uri.params(params))
 
+  def pushdownToFilters[F](
+      sparkFilters: Array[Filter],
+      mapping: Map[String, String] => F,
+      allFilter: F): (Vector[CogniteId], Vector[F]) = {
```

Comment:
Can you think of a better name than `allFilter`? (I couldn't)

Reply:
Stanislav in the next 3 days 😆 
![image](https://user-images.githubusercontent.com/92026183/146516932-f83ea5ba-1b0a-4125-8ac4-744750f64fa4.png)
Just kidding